FirebaseMessaging/Sources/Token/FIRMessagingTokenManager.h

```diff
@@ -173,4 +171,10 @@ typedef NS_OPTIONS(NSUInteger, FIRMessagingInvalidTokenReason) {
  */
 - (void)saveDefaultTokenInfo:(NSString *)defaultFcmToken;
 
+/*
+ * Sets the default FCM token the first time from FIRMessaging.
+ *
+ */
+- (void)setDefaultFCMToken:(NSString *)defaultFcmToken;
```

Comment:
This method looks confusingly similar to the one defined above `- (void)saveDefaultTokenInfo:(NSString *)defaultFcmToken;`. What is the difference between them? Do we actually need both?

Reply:
They did lost meaning after the refactor. I updated the description and naming of the methods.

Reply:
It still looks a bit confusing to me, sorry if I'm missing something. As far as I can see the method `- (void)setDefaultFCMToken:(NSString *)defaultFcmToken` only posts as notification about the token update. Maybe a name like `defaultFCMTokenWasUpdate:` or `postDefaultFCMTokenRefreshNotification:` would better reflect the implementation?

Reply:
Yep, this function eventually trigger a method by the notification to actually set the token, but I see the naming is confusing. Rename it to postDefaultFCMTokenRefreshNotification:.